file-formats/intf/zif_intf_v1.intf.abap

```diff
@@ -0,0 +1,29 @@
+INTERFACE zif_intf_v1 PUBLIC.
+
+  TYPES: BEGIN OF header,
```

Comment:
it looks like this is an old file that made it into this branch? filename in main is `zif_aff_intf_v1.intf.abap`

Reply:
already started to be worried. Yep, this file does no longer belong to the repo